homeassistant/components/motion_blinds/cover.py

```diff
@@ -266,44 +270,53 @@ async def async_scheduled_update_request(self, *_):
             self._previous_positions = []
             self._requesting_position = False
 
-    def request_position_till_stop(self):
+    async def async_request_position_till_stop(self):
         """Request the position of the blind every UPDATE_INTERVAL_MOVING seconds until it stops moving."""
         self._previous_positions = []
         if self._requesting_position or self.current_cover_position is None:
             return
 
         self._requesting_position = True
-        track_point_in_time(
+        async_track_point_in_time(
```

Comment:
You could use async_call_later

Reply:
thanks that simplifies, I just pushed a commit using async_call_later.